.gitignore

```diff
@@ -2,5 +2,5 @@ __pycache__
 *.pyc
 */.DS_Store
 .cache
-.swp
+*.swp
```

Comment:
You're doing this in the other pr now right?

Reply:
Oh yea, i think i accidentally committed to both